files/image_config/warmboot-finalizer/finalize-warmboot.sh

```diff
@@ -101,16 +101,30 @@ function check_list()
     echo ${RET_LIST}
 }
 
+function set_cpufreq_governor() {
+    local -r governor="$1"
+    echo "$governor" | tee /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor 1> /dev/null \
```

Comment:
Will this setting (or the way we are writing to a file) be effective at runtime?

Reply:
Yes